lib/modules/Logger.js

```diff
@@ -1,6 +1,17 @@
 export default store => next => action => {
-  console.log('dispatching', action);
-  const result = next(action);
-  console.log('next state', store.getState());
+  let result;
+  console.group(action.type);
```

Comment:
welcome to the team! also, should this fall within the `if (console.group)` clause?